hooks/pre_commit/13-tuning.rb

```diff
@@ -19,7 +19,9 @@
     new_tuning = current_tuning
   end
 
-  unless app_value(:disable_system_checks)
+  if app_value(:disable_system_checks)
+    logger.warn 'Skipping system checks.'
+  else
```

Comment:
This change is here so we don't lose the warning a user is disabling system checks?